fusesoc/coremanager.py

```diff
@@ -227,6 +227,8 @@ def find_cores(self, library):
                     except ImportError as e:
                         w = 'Failed to register "{}" due to unknown provider: {}'
                         logger.warning(w.format(core_file, str(e)))
+                    except ValueError as e:
+                        logger.warning(e)
```

Comment:
When does that happen, and why is it harmless? A comment would be great here.

Reply:
Yeah.... about that... I haven't got a clue anymore what that was supposed to fix. Better to drop it until we figure it out